azure/src/azure-account.api.d.ts

```diff
@@ -4,61 +4,61 @@
  *--------------------------------------------------------------------------------------------*/
```

Comment:
nit: revert the whitespace changes in this file.

Reply:
I think the spaces were wrong originally. Everything is indented twice for some reason.

Reply:
It was tabs before, IIRC, and every time we get near it, it's trying to change it to spaces. Not even worth fighting with it...